File: hello-world/src/test/java/kz/sabyr/quarkus/helloworl/service/MockMyService.java
package kz.sabyr.quarkus.helloworl.service;

import io.quarkus.test.Mock;
import kz.sabyr.quarkus.service.MyService;

import javax.enterprise.context.ApplicationScoped;

@Mock
@ApplicationScoped
public class MockMyService implements MyService {
    @Override
    public String generateUUID() {
        return "1234";
    }
}

File: hello-world-kotlin/src/test/kotlin/kz/sabyr/quarkus/helloworlkotlin/service/MockMyService.kt
package kz.sabyr.quarkus.helloworlkotlin.service

import io.quarkus.test.Mock
import javax.enterprise.context.ApplicationScoped

@Mock
@ApplicationScoped
class MockMyService: MyService {
    override fun generateUUID(): String {
        return "1234"
    }
}